simbad/rotsearch/phaser_search.py

```diff
@@ -255,6 +261,23 @@ def summarize(self, csv_file):
         summarize_result(self.search_results,
                          csv_file=csv_file, columns=columns)
 
+    @staticmethod
+    def calculate_ermsd(nres, id):
+        """Calculate the estimated rmsd based on the number of residues and sequence id"""
+
+        if id < 0 or id > 100:
+            msg = "Sequence id must be a number between 0 and 100"
+            raise RuntimeError(msg)
+
+        if nres < 0:
+            msg = "The number of residues must be greater than 0"
+            raise RuntimeError(msg)
+
+        return np.around((1.54967686e+00 + 1.69034597e-03 * nres + -2.57055256e-02 * id + -5.92896517e-07 * nres ** 2
```

Comment:
Is this the equation they use for the eRMSD?

Reply:
No, they just give the graph of values from which I plotted the points and fitted a curve to.